ui/app/templates/permission_scheduled_changes.html

```diff
@@ -41,12 +41,12 @@
 
     <h3 class="panel-title">
       <div style="float: right">
-        <i ng-if=" scheduled_changes_rules_count && $first && (currentPage == 1)">Current</i>
```

Comment:
I don't think you want to change the ng-if on this one -- it already has the correct conditions for whether or not it should be displayed. Same thing on the other pages.

Reply:
So the _Current_ should be showing?